### ISSUE TITLE
Auto-generate testable demos from all pull requests with GitPod.io

### ISSUE THREAD

**Issue by jywarren** [publiclab/PublicLab.Editor#475]
To make testing out new PRs faster and easier, we could integrate with GitPod.io, which auto-builds and hosts a browser-based version. I think, though i'm not sure, that we could do this with IS:

https://www.gitpod.io/

This could make debugging much easier too!

**Comment by Shulammite-Aso** [comment]
I'm reading more about this,  and so far it looks like something that will do good to our development process, in terms of both efficiency and time management.

I'm going on to read further.

**Comment by keshav234156** [comment]
Seems interesting. Reading more about this!!!

**Comment by Shulammite-Aso** [comment]
I was trying to familiarize myself with gitpod when i had some issues with running my project on the browser to see my changes,  so I reached out to the gitpod guys for help, and as part of the assistance, they've just made a pull request with some configuration to get started with gitpod support. At #483 

I'm thinking this would be helpful for when we're set to add gitpod to the project.

**Comment by jywarren** [comment]
Hi @Shulammite-Aso and all! We just completed this at https://github.com/publiclab/image-sequencer/pull/1668 and are partway through in https://github.com/publiclab/Leaflet.DistortableImage/pull/680

I enabled it for all publiclab repos, so i believe it's just getting the `.gitpod.yml` file formatted correctly! I think it'd be great and would make testing out fixes in a live editor on each PR really easy. I just did this in a complex PR in `image-sequencer` and it was amazing to be able to take the PR for a spin so quickly! 👍  doing this and I can hopefully help out.
